python_modules/dagster/dagster/_core/storage/event_log/schema.py

```diff
@@ -137,6 +137,25 @@
     db.Column("create_timestamp", db.DateTime, server_default=get_current_timestamp()),
 )
 
+AssetChecksTable = db.Table(
+    "asset_checks",
+    SqlEventLogStorageMetadata,
+    db.Column(
+        "id",
+        db.BigInteger().with_variant(sqlite.INTEGER(), "sqlite"),
+        primary_key=True,
+        autoincrement=True,
+    ),
+    db.Column("name", db.Text),
+    db.Column("asset_key", db.Text),
+    db.Column("status", db.Text),
```

Comment:
Per https://www.notion.so/dagster/Asset-checks-implementation-9aafff00bc1a4545954d42ef4952bc69?pvs=4#5b117407ae2847409d673a3d67f09b54, this holds SUCCESS, FAIL, PLANNED

Reply:
Skimming this, it's not clear to me whether this is talking about the success/failure of the check step vs. the success of the check itself.  Can we find a way to make it more clear? If it's the latter, the column should probably have a name that references the relevant property on `AssetCheckResult`.